src/pygeosx/unitTests/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+
+
+add_custom_target( pygeosx_unit_tests
+                   COMMAND ${CMAKE_BINARY_DIR}/lib/PYGEOSX/bin/python3 -m pip install pytest --quiet --disable-pip-version-check
```

Comment:
I do not think that the installation should be done there, shouldn't it?
Don't you end up trying to trigger the installation everytime?

Reply:
Long term, no, I don't think we should be doing this here.  For systems that already have pytest installed, pip just silently checks that the required packages are installed and moves on.

Right now, we are installing pygeosx by:

1) Creating a virtualenv off of the base python distribution (bin/lib/PYGEOSX)
2) Manually linking the pygeosx/pylvarray libraries in the PYGEOSX env

Since we aren't using the typical setup process for a package, we aren't enforcing required packages anywhere (aside from build errors if required python/package headers aren't present).  Any thoughts?  Also, are we targeting any systems where pip install is not viable?

Reply:
I would say that it's not GEOSX's responsibility to provide a way to install `pytest`.
Furthermore, there are so many ways to build envs in python that even imposing `pip` may not be right.

Reply:
@TotoGaz - I've removed the pip command here, so this is now up to the user